### DISCUSSION TITLE
USB Trigger

### DISCUSSION THREAD

**Discussion by Tropicfan** [gopro/labs#253] [General]
Is there any way to have the camera switched off as USB power drops?

I'm using my camera in my truck and am looking for a way to automatically shut it down when leaving the truck

I'm using USB Trigger "on" to boot the camera as condition to get it reacting to the remote control but it should not stay on draining the battery when not driving

**Comment by dnewman-gpsw** [comment]
This should works as you want https://gopro.github.io/labs/control/usb/, it will stop recording and turn off the camera when USB power is removed.  If you doing addition things with the remote control, the new Labs firmware in a few weeks expand your options.

**Comment by Tropicfan** [comment]
Hi David,
Thanks for your hint
As I just want to switch on the camera, I removed the "!S" - so it just switches on without starting to capture.
Removing the ext. power, the camera still reacts to the remote controle, so I think it does not switch off, does it?
Best regards
Holger

⁣Obter o BlueMail para Android ​

Em 4 de mar. de 2023 02:10, em 02:10, David Newman ***@***.***> escreveu:

**Reply by dnewman-gpsw** [reply]
It is still not clear what the concern is. Camera will not stay on indefinitely, unless you ask them too.  There are multiple was to make them auto boot upon USB power, although some a camera model dependent.

**Comment by Tropicfan** [comment]
Mine is a Hero8 Black

In configuration menu of the camera I asked it to never shut down, because if it were shut down, there would be no way to connect to remote control to start or stop capturing.
As I asked to never shut down automatically, it will stay on even if USB power drops, draining the battery.
That's why I'm looking for a way to have it shut down with USB connection being cut off
Best regards
Holger 

⁣Obter o BlueMail para Android ​

Em 4 de mar. de 2023 14:23, em 14:23, David Newman ***@***.***> escreveu:

**Comment by Tropicfan** [comment]
Additional information:
I'm using my camera in my Truck - powered by USB charger - not really as a dash cam, but just to capture nice moments of the trip, manually starting capture with the remote control 

⁣Obter o BlueMail para Android ​

Em 4 de mar. de 2023 16:08, em 16:08, Holger Jacoby ***@***.***> escreveu:

**Reply by dnewman-gpsw** [reply]
This is the sort of complex capture scenarios the next Labs command set is designed to help with. It is out in a few weeks for HERO11, but not sure if there will be any back-porting available (engineering and QA time needed, for H10, H9, MAX, and H8 etc.)  HERO8 has no turn off on USB off command. The closest would be to just using !MWAKE=2, with no battery installed. The camera will boot on USB Power, but be in standby, auto connect to the remote if present. Now it will work as you plan, except a USB power off will be abrupt, and if recording at the time, the file will need repair upon the next start.  Give that a try.

**Comment by Tropicfan** [comment]
Got it - thanks for your patience
Seems a usable solution 🙂

Maybe there will be any back-porting available in the future - Hero8 is marvelous and deserves it 

Best regards
Holger

⁣Obter o BlueMail para Android ​

Em 4 de mar. de 2023 17:28, em 17:28, David Newman ***@***.***> escreveu:

**Comment by Tropicfan** [comment]
Last question:
Does not hurt the camera operating it without battery?
Best regards
Holger

⁣Obter o BlueMail para Android ​

Em 4 de mar. de 2023 20:16, em 20:16, Holger Jacoby ***@***.***> escreveu:

**Reply by dnewman-gpsw** [reply]
Not at all.  Only downside, with no battery for more than 12 hours, can lose date/time.  Harmless otherwise.  I never keep batteries in my cameras.
